Makefile

```diff
@@ -54,12 +54,16 @@ DOCKER_ENVS := \
 BIND_DIR := $(if $(BINDDIR),$(BINDDIR),$(if $(DOCKER_HOST),,bundles))
 DOCKER_MOUNT := $(if $(BIND_DIR),-v "$(CURDIR)/$(BIND_DIR):/go/src/github.com/docker/docker/$(BIND_DIR)")
 
+# This allows the test suite to be able to run without worrying about the underlying fs used by the container running the daemon (e.g. aufs-on-aufs), so long as the host running the container is running a supported fs.
+# The volume will be cleaned up when the container is removed due to `--rm`.
+DOCKER_BUNDLES_VOLUME := -v /go/src/github.com/docker/docker/bundles
```

Comment:
`BIND_DIR=bundles` only if `$DOCKER_HOST` is not set, so there should be no effect there.
`docker/docker` was intentional, but yeah to avoid copying I'll do `/bundles`.